src/System.Runtime.TimeZoneData/System.Runtime.TimeZoneData.csproj

```diff
@@ -18,7 +18,13 @@
     <Area Include="backward" />
   </ItemGroup>
 
-  <Target Name="CheckVersion">
+  <Target Name="SetLatestVersion" DependsOnTargets="GetLatestVersion">
+    <PropertyGroup>
+      <TimeZoneDataVersion>$(LatestTimeZoneDataVersion)</TimeZoneDataVersion>
+    </PropertyGroup>
+  </Target>
```

Comment:
I would recommend to remove that target (mostly for simplicity) and move the propertygroup into the `UpdateToLatestVersion` target and change the DependsOnTargets attribute there.

Reply:
The property update wasn't getting updated in the update and compile steps until I moved it into a separate target.

Reply:
I still haven't figured out how to avoid the separate target but now the separation makes more sense.